content/techniques/configuration.md

```diff
@@ -317,11 +317,13 @@ export class AppService {
 
 The `@nestjs/config` package allows you to take advantage of the **"Expandable Variables"** functionality. You can use your exisiting `.env` variables to declare new variables within `.env` itself. For eg:
```

Comment:
I suggest changing this
"The `@nestjs/config` package allows you to take advantage of the **"Expandable Variables"** functionality. You can use your exisiting `.env` variables to declare new variables within `.env` itself. For eg:"
to
"The `@nestjs/config` package  supports environment variable expansion.  With this technique, you can create nested environment variables, where one variable is referred to within the definition of another.  For example:"

Reply:
@johnbiundo Done 👍